public/home.html

```diff
@@ -8,11 +8,18 @@ <h1 class="mb-1 fw-semibold">
     </div>
   </div>
 </div>
+<script>
+  function copy() {}
+</script>
 
 <div class="container w-95 text-center text-muted">
   <div id="blockchain-metrics"></div>
 </div>
 
+<div class="container w-95 text-center text-muted">
+  <div onclick="copy()" id="rpc-url"></div>
```

Comment:
```suggestion
  <div class="cursor-grab" onclick="copy()" id="rpc-url"></div>
```
https://tailwindcss.com/docs/cursor

Reply:
I realize it's bootstrap 😵. could use `role="button"` to get a `pointer` cursor, but i guess the style tag will have to do if you want grab

Reply:
I like using the class vs id, I'll do that.